tests/conftest.py

```diff
@@ -24,4 +26,18 @@ def synapse_manifest():
 
 @pytest.fixture()
 def local_manifest():
-    return pd.read_csv("tests/data/mock_manifests/local_manifest.csv")
\ No newline at end of file
+    return pd.read_csv("tests/data/mock_manifests/local_manifest.csv")
+
+
+@pytest.fixture()
+def schema_explorer():
```

Comment:
Could this be repurposed as a helper function like `get_data_frame()`, but something like `get_schema_explorer()`? Because if this is only every used in testing, the JSON-LD path will always be relative to `tests/data/`.